Resources/doc/reference/advanced_usage.rst

```diff
@@ -38,3 +38,15 @@ core routes):
     sonata_page:
         page_defaults:
             homepage: {decorate: false} # disable decoration for homepage route_name
+
+Templates customization
+-----------------------
+
+You can also customize/overwriting the templates of the bundle.
```

Comment:
Sometimes you prefer use inheritance and extends original template to ease vendor updates and only modify part of templates. Should I delete this part ?